modules/machine-health-checks-pausing.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+
+// * updating/updating-cluster-cli.adoc
+// * updating/updating-cluster-between-minor.adoc
+// * updating/updating-restricted-network-cluster.adoc
+
+[id="machine-health-checks-pausing_{context}"]
+= Pausing a MachineHealthCheck resource
+
+During the upgrade process, nodes in the cluster might become temporarily unavailable. In case of worker nodes, the machine health check might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, pause all the `MachineHealthCheck` resources before updating the cluster.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+.Procedure
+
+1. To list all the available `MachineHealthCheck` resources that you want to pause, run the following command:
```

Comment:
These enumerations shouldn't be hardcoded. I would suggest using `. ` for an ordered list here and on line 25.